apps/base-docs/docs/building-with-base/guides/hyperframes.md

```diff
@@ -0,0 +1,327 @@
+---
+title: 'HyperFrames'
+description: Learn how to make cross-linked hyperframes in an organized manner.
+hide_table_of_contents: false
+---
+
+[Frames] on [Farcaster] are getting more complex. Developers are now building interactions that require a handful, or even dozens of frames in response to various user states, inputs, and actions. HyperFrames are a system to organize navigation for large numbers of frames, using [OnchainKit]. In this tutorial, we'll use making the navigation for an [old-school adventure game] fully in a frame. You can use this same technique for many other intents with your Frames, such as games, stores, customized mints, etc.
+
+:::caution
+
+Frames are brand new and tools are evolving quickly. Check the links above for changelogs!
+
+:::
+
+---
+
+## Objectives
+
+By the end of this lesson, you should be able to:
+
+- Build hyperframes - connected [Farcaster] Frames that manage significant numbers of cross-linked frames in an organized fashion
+- Add management of conditional states to the navigation system
+
+---
+
+## Prerequisites
+
+### Vercel
+
+You'll need to be comfortable deploying your app to [Vercel], or using another solution on your own. Check out our tutorial on [deploying with Vercel] if you need a refresher!
+
+### Farcaster
+
+You must have a [Farcaster] account with a connected wallet. Check out the [Base channel] to stay in the loop when we release guides like this!
+
+### Frames
+
+You should be comfortable with the basics of creating Farcaster [Frames]. If you aren't, check out our tutorial on [NFT Minting Frame].
+
+---
+
+## Getting Started
+
+Start by creating a new Frames project with Next.js. Use [a-frame-in-100-lines] if you want a jump start!
+
+You'll also need to create or find a handful of images to use for each frame. AI tools are wonderful for this type of prototyping, or you can right-click and save the images from our demo, linked above.
```

Comment:
`or you can right-click and save the images from our demo, linked above` -

Is this referring to the `a-frame-in-100-lines` repo?